src/recEventTracker.cpp

```diff
@@ -341,8 +342,8 @@ void RecEventTracker::saveOfflineEventLog(uint64 timestampFrom, uint64 timestamp
 // below is the version that creates a csv
 /*
 void RecEventTracker::saveOfflineEventLog(uint64 timestampFrom, uint64 timestampTo, const QString& fileName) {
-    delim = applicationSettings->value("delimiterToUse", ",").toString()[0];
-    //delim = applicationSettings->value("delimiterToUse", ',').toChar(); // somehow this just doesnt work
+    delim = applicationSettings->value("dataWriterDelimiterToUse", ",").toString()[0];
```

Comment:
This is commented out but I believe the variable is DataWriterDelimiter, not dataWriterDelimiterToUse. Probably should just delete the function if it's commented out anyway

Reply:
Done